server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -116,7 +116,7 @@ private void registerAsyncMetrics(MeterRegistry registry) {
 
         metrics.add(
             registry.registerLongGauge(
-                "es.node.stats.indices.translog.operations",
+                "es.node.stats.indices.translog.operations.count",
                 "Number of transaction log operations.",
                 "operation",
```

Comment:
Should this change to "count"?

Reply:
translong.operations.count feels addivite to me. perhaps it should be an async counter instead

Reply:
I have changed this to total.
unit here is misleading. we often use just `count` as counters are often unit-less.
I wonder if we should not  allow empty strings here